templates/site/node.html

```diff
@@ -10,6 +10,8 @@
                         <!--H3 title-->
                         <div id="affiliation" class="mb-1 text-body-secondary"></div>
                         <div id="description"></div>
+                        <div id="registration_status" class="mt-3"></div>
```

Comment:
Move this lower down, it should at the very least be after the link to the node's landing page, probably after all of the node information but before the services.

Reply:
Moved the registration status and link divs to after the node information and before services table.